python/llm/test/inference/test_optimize_model.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import os
+
+from bigdl.llm.transformers import AutoModelForCausalLM, AutoModel
+from transformers import LlamaTokenizer, AutoTokenizer
+
+
+llama_model_path = os.environ.get('LLAMA_ORIGIN_PATH')
+bloom_model_path = os.environ.get('BLOOM_ORIGIN_PATH')
+chatglm2_6b_model_path = os.environ.get('ORIGINAL_CHATGLM2_6B_PATH')
+replit_code_model_path = os.environ.get('ORIGINAL_REPLIT_CODE_PATH')
+
+prompt = "Once upon a time, there existed a little girl who liked to have adventures. She wanted to go to places and meet new people, and have fun"
+
+@pytest.mark.parametrize("Model, Tokenizer, model_path, prompt", [
```

Comment:
This parameterize is awesome!

Reply:
For future reference, it is possible to customize the test identifier. 

Currently, each test's identifier is [Model-Tokenizer-model_path, prompt]
> python/llm/test/inference/test_optimize_model.py::test_optimize_model[AutoModelForCausalLM-LlamaTokenizer-/home/runner/_work/BigDL/BigDL/../llm/origin-models/llama-7b-hf-Once upon a time, there existed a little girl who liked to have adventures. She wanted to go to places and meet new people, and have fun] PASSED

We can predefine or customize the test identifier, for example,
```
@pytest.mark.parametrize("Model, Tokenizer, model_path, prompt", [
    (AutoModelForCausalLM, LlamaTokenizer, llama_model_path, prompt),
    (AutoModelForCausalLM, AutoTokenizer, bloom_model_path, prompt),
    (AutoModel, AutoTokenizer, chatglm2_6b_model_path, prompt),
    (AutoModelForCausalLM, AutoTokenizer, replit_code_model_path, prompt)],
    ids=['llama','bloom','chatglm2_6b','replit']
)
```
to get this
> python/llm/test/inference/test_optimize_model.py::test_optimize_model[llama] PASSED